boilerplate-swiftui-bloc/Configurations/Configs.swift

```diff
@@ -7,8 +7,10 @@
 
 import Foundation
 
-class Configs {
+final class Configs {
     static let shared = Configs()
     
-    var isStorybook: Bool { false }
+    var isStorybook: Bool {
+        AppEnvironment.isStorybook
```

Comment:
if we using computed property here, is that it will read from Bundle every time ? If we have singleton to store all needed configs, we should read it once when app launch.